python/server/qgsmsutils.sip

```diff
@@ -0,0 +1,34 @@
+/***************************************************************************
+    qgsmsutils.h
+    ---------------------
+    begin                : August 2010
+    copyright            : (C) 2010 by Marco Hugentobler
+    email                : marco dot hugentobler at sourcepole dot ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+%ModuleHeaderCode
+#include <QString>
+%End
+
+//! Some utility functions that may be included from everywhere in the code
+namespace QgsMSUtils
```

Comment:
With QGIS 3.0 we are trying to follow Qt's name capatilization conventions, so this should be `QgsMsUtils`. But we're also trying to remove all abbreviated names so `QgsMapServerUtils` would be best.